doc/CONTRIBUTING.md

```diff
@@ -85,10 +85,28 @@ Latest stabe release resides in branch `main`, while pre-releases (alpha,
 beta, release candidates etc.) reside in branch _prerelease_. Late fixes in
 past stabe releases for maintenance, if any, are found in _legacy_ branches.
 
-Contribution admission policy requires developers to propose modifications
-as feature branches and submit them as pull requests, which in turn
-need to be approved by other developers before being merged into the proper 
-branches.
+It's important that every pull request be linked to an issue. You can pick
+an issue that already exists or create one yourself. After an issue has been
+selected, assign yourself to it to notify other collaborators that you are
```

Comment:
```suggestion
selected, assign yourself so other collaborators can be aware that you are
```

Reply:
Hey thanks for all your suggestions, I think they help to make the text clearer. 
Just be aware that you can make one single suggestion for multiple successive lines, making it easier to apply it. Take a look in step 4 of [this document](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request).